streams-bootstrap-core/src/testFixtures/java/com/bakdata/kafka/TestUtil.java

```diff
@@ -24,21 +24,13 @@
 
 package com.bakdata.kafka;
 
-import static net.mguenther.kafka.junit.EmbeddedKafkaCluster.provisionWith;
-import static net.mguenther.kafka.junit.EmbeddedKafkaClusterConfig.newClusterConfig;
-import static net.mguenther.kafka.junit.EmbeddedKafkaConfig.brokers;
-
 import lombok.experimental.UtilityClass;
-import net.mguenther.kafka.junit.EmbeddedKafkaCluster;
+import org.testcontainers.kafka.ConfluentKafkaContainer;
+import org.testcontainers.utility.DockerImageName;
 
 @UtilityClass
 public class TestUtil {
-    public static EmbeddedKafkaCluster newKafkaCluster() {
-        return provisionWith(newClusterConfig()
-                .configure(brokers()
-                        .with("transaction.state.log.num.partitions", 10)
-                        .with("offsets.topic.num.partitions", 10)
-                        .build())
-                .build());
+    public static ConfluentKafkaContainer newKafkaCluster() {
+        return new ConfluentKafkaContainer(DockerImageName.parse("confluentinc/cp-kafka:7.6.0"));
```

Comment:
Did you already look into the native image?

Reply:
Apache image was first published for 3.7. Native image for 3.8. Will use them when upgrading